### PR TITLE
WEBUI-33: add crowdin synchronization workflow

### PR DESCRIPTION
**Pull request by richardsd**
This is a work in progress:
- First solution is using the https://github.com/nuxeo/tools-nuxeo-crowdin/ to perform the work (Mimics the jenkins job that regularly triggers the crowdin sync)
- Second approach: use Crowdin language mapping to solve our problem with locales
- Third approach: use a configuration file to solve the language mapping issue

After some discussion with the team, we agreed that the third approach seems the best one.

### CODE REVIEW

File: .github/workflows/crowdin.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  #   branches:
+  #     - maintenance-3.0.x
+  #   paths:
+  #     - 'i18n/messages.json'
+
+  # XXX for testing only
+  # pull_request:
+  #   branches:
+  #     - maintenance-3.0.x
+  workflow_dispatch:
```
Comment (semisse) [inline .github/workflows/crowdin.yaml]: should this be commented as well?

File: .github/workflows/crowdin.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Workflow that syncronizes translations from Crowdin
+name: Crowdin Sync
+
+on:
+  # Check for updates every day
```
Comment (richardsd) [inline .github/workflows/crowdin.yaml:5]: About scheduling the Crowdin sync:
- every day at it happens nowdays with the Jenkins job
- on push to `maintenance-3.0.x` when a new label is added to `messages.json`

Wdyt?

**Review by semisse — DISMISSED** [review]

File: .github/workflows/crowdin.yaml (hunk 1 above)
Comment (richardsd) [inline .github/workflows/crowdin.yaml]: To be removed.

File: .github/workflows/crowdin.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          project_id: ${{ secrets.CROWDIN_PROJECT_ID }}
+          token: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+          # Load external configuration file (to support translation rename)
+          config: crowdin-conf.yml
+
+          # Auto-approve reference sentences we are pushing (english)
+          auto_approve_imported: true
+
+          # Name of the branch where to merge the translations
+          localization_branch_name: WEBUI-33-crowdin-gh-actions
```
Comment (richardsd) [inline .github/workflows/crowdin.yaml]: The branch should be `maintenance-3.0.x`.

File: .github/workflows/crowdin.yaml (hunk 1 above)
Comment (richardsd) [reply .github/workflows/crowdin.yaml]: to be removed, just here to support code review.
Can you give me some feedback about the crowdin schedule I posted bellow? thanks

**Review by Gabez0r — COMMENTED** [review]
It's looking good! 👏 💪

File: .github/workflows/crowdin.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          config: crowdin-conf.yml
+
+          # Upload sources for translation
+          upload_sources: true
+
+          # Auto-approve reference sentences we are pushing (english)
+          auto_approve_imported: true
+
+          # Name of the branch where to merge the translations
+          localization_branch_name: WEBUI-33-crowdin-gh-actions
```
Comment (Gabez0r) [inline .github/workflows/crowdin.yaml]: This must be updated, right?

**Review by Gabez0r — COMMENTED** [review]
LGTM 👍

File: .github/workflows/crowdin.yaml (hunk 4 above)
Comment (richardsd) [reply .github/workflows/crowdin.yaml]: Yes, it's going to be `maintenance-3.0.x`.

**Review by semisse — DISMISSED** [review]

**Review by Gabez0r — DISMISSED** [review]

File: .github/workflows/crowdin.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# Workflow that syncronizes translations from Crowdin
+name: Crowdin Sync
+
+on:
+  # Check for updates every day
+  schedule:
+    - cron: '0 0 * * *'
```
Comment (Gabez0r) [inline .github/workflows/crowdin.yaml:7]: Same comment https://github.com/nuxeo/nuxeo-elements/pull/549#discussion_r913649378

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

File: .github/workflows/crowdin.yaml (hunk 5 above)
Comment (richardsd) [reply .github/workflows/crowdin.yaml:7]: Same comment as here https://github.com/nuxeo/nuxeo-elements/pull/549#discussion_r913752099

**Pull request merged**